lib/bemxjst/tree.js

```diff
@@ -110,9 +109,7 @@ ExtendMatch.prototype.wrapBody = function wrapBody(body) {
       for (var i = 0; i < keys.length; i++)
         changes[keys[i]] = body[keys[i]];
 
-      return local(changes)(function preApplyCtx() {
-        return applyCtx(this.ctx);
```

Comment:
It’s not about droping `local` from bem-xjst. It’s about use second argument in `applyCtx()` only here.